docs/source/api/algorithms/graphs/jaccard-similarity.rst

```diff
@@ -0,0 +1,45 @@
+.. ******************************************************************************
+.. * Copyright 2021 Intel Corporation
+.. *
+.. * Licensed under the Apache License, Version 2.0 (the "License");
+.. * you may not use this file except in compliance with the License.
+.. * You may obtain a copy of the License at
+.. *
+.. *     http://www.apache.org/licenses/LICENSE-2.0
+.. *
+.. * Unless required by applicable law or agreed to in writing, software
+.. * distributed under the License is distributed on an "AS IS" BASIS,
+.. * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. * See the License for the specific language governing permissions and
+.. * limitations under the License.
+.. *******************************************************************************/
+
+.. default-domain:: cpp
+
+.. _api_jaccard_similarity:
+
+===================================
+Jaccard Similarity
+===================================
+
+.. /use the algorithm name/
+.. include:: ../../../includes/graphs/graph-algorithm-introduction.rst
+
+------------------------
+Mathematical formulation
+------------------------
+
+Refer to :ref:`Developer Guide: Jaccard Similarity <alg_jaccard_similarity>`.
```

Comment:
I don't see this topic in Developer Guide.

There's a warning in docbuild: `WARNING: undefined label: alg_jaccard_similarity`